provisioner/terraform/resources.go

```diff
@@ -883,10 +884,24 @@ func ConvertState(ctx context.Context, modules []*tfjson.StateModule, rawGraph s
 			)
 		}
 
+		if len(preset.Prebuilds) != 1 {
```

Comment:
Will this not warn incorrectly when there are no prebuilds defined at all (which is perfectly valid)?

Shouldn't this be `len(preset.Prebuilds) > 1`?